rstream/client.py

```diff
@@ -361,9 +361,9 @@ async def close(self) -> None:
                     )
 
                 except asyncio.TimeoutError:
-                    logger.warning("timeout in client close() sync_request:")
-                except BaseException as exc:
-                    logger.exception("exception in client close() sync_request", exc)
+                    logger.warning("timeout in client close() sync_request")
+                except BaseException:
+                    logger.exception("exception in client close() sync_request")
```

Comment:
Instead of completely removing the exception wouldn't be better to add the formatter %s like you did in consumer.py 
logger.warning("exception in unsubscribe of Consumer: %s", exc)?